dev-support/ci/selective_ci_checks.sh

```diff
@@ -190,10 +192,13 @@ function check_if_tests_are_needed_at_all() {
 function run_all_tests_if_environment_files_changed() {
     start_end::group_start "Check if everything should be run"
     local pattern_array=(
-        "^.github/workflows/"
+        "^.github/workflows/post-commit.yml"
```

Comment:
Does this mean we are only including the `post-commit.yml` rather than all the files under `.github/workflows/` with the change?